etc/profile-m-z/postman.profile

```diff
@@ -0,0 +1,22 @@
+# Firejail profile for postman
+# Description: API testing platform
+# This file is overwritten after every install/update
+# Persistent local customizations
+include postman.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/Postman
+noblacklist ${HOME}/Postman
+
+mkdir ${HOME}/.config/Postman
+mkdir ${HOME}/Postman
+whitelist ${HOME}/.config/Postman
+whitelist ${HOME}/Postman
+
+#private-opt postman
```

Comment:
(This is the line used on microsoft-edge.profile as mentioned in #5307)

Does Postman work with this change?

That is, do you still get the following error?

> ```
> Error: no suitable /usr/bin/postman executable found
> ```

Reply:
Sorry, I hadn't copied the code suggestion itself; edited it into the previous
comment.

Cc: @Kobaxidze256

Reply:
It seems to be working.
@glitsj16 create a commit with your profile.

Reply:
Oh wait, that's not how it works.